release/release_logs/1.3.0/xgboost_tests/ft_small_elastic.txt

```diff
@@ -0,0 +1,220 @@
+Script started on 2021-03-30 04:50:46-07:00 [TERM="screen" TTY="/dev/pts/11" COLUMNS="80" LINES="24"]
```

Comment:
hmm does it make sense to save this too?

Reply:
Happy to remove these if you think this is unnecessary information - we're not collecting performance measures here, so I guess it's not that important to keep them